### PR TITLE
[DSS/DSM] .gitignore file modified

### PR DESCRIPTION
**Pull request by Trimax**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -107,3 +107,9 @@ tcell/logs/
 *.conf
 !study-builder/**/*.conf
 /appengine/deploy/
+
+# Output and target directories
+**/output/
+**/target/
+**/output-config/
+**/output-build-config/
```
Comment (bskinner) [inline .gitignore]: The leading `**/` is not necessary

File: pepper-apis/.gitignore [hunk 2]
```diff
@@ -18,15 +18,16 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
-target/
+# Output and target directories
+**/output/
+**/target/
+**/output-config/
+**/output-build-config/
 surefire-report.html
```
Comment (bskinner) [inline pepper-apis/.gitignore]: The leading `**/` is not necessary

**Review by bskinner — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Trimax) [reply .gitignore]: Removed

File: pepper-apis/.gitignore (hunk 2 above)
Comment (Trimax) [reply pepper-apis/.gitignore]: Removed

File: pepper-apis/.gitignore [hunk 3]
```diff
@@ -18,15 +18,18 @@
...
 
 # rendered secrets
 config/*.conf
 !config/waf.conf
-output/
-output-config/
-output-build-config/
 docker-compose.yaml
 appengine/deploy/*.conf
```
Comment (bskinner) [inline pepper-apis/.gitignore:29]: Modifications to this file aren't necessary

Comment (Trimax) [reply pepper-apis/.gitignore:29]: Well, we now can simplify it since the ignore options moved up

**Review by bskinner — APPROVED** [review]

**Pull request merged**